.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: java
+sudo: false
```

Comment:
What's this for? Was this left in from something else - I know you talked about doing this in travis but I can't remember why and it doesn't seem relevant here.